public/privacy-policy.html

```diff
@@ -1,161 +1,258 @@
-    be directed to that third party's site. We strongly advise you to review the Privacy Policy of every site you
-    visit.</p>
-<p>We have no control over and assume no responsibility for the content, privacy policies or practices of any third
-    party sites or services.</p>
-
+<p>
+    Our Service may contain links to other sites that are not operated by us. If
+    you click a third party link, you will be directed to that third party's
+    site. We strongly advise you to review the Privacy Policy of every site you
+    visit.
+</p>
+<p>
+    We have no control over and assume no responsibility for the content,
+    privacy policies or practices of any third party sites or services.
+</p>
 
 <h2>Children's Privacy</h2>
 <p>Our Service does not address anyone under the age of 18 ("Children").</p>
-<p>We do not knowingly collect personally identifiable information from anyone under the age of 18. If you are a parent
-    or guardian and you are aware that your Child has provided us with Personal Data, please contact us. If we become
-    aware that we have collected Personal Data from children without verification of parental consent, we take steps to
-    remove that information from our servers.</p>
-
+<p>
+    We do not knowingly collect personally identifiable information from anyone
+    under the age of 18. If you are a parent or guardian and you are aware that
+    your Child has provided us with Personal Data, please contact us. If we
+    become aware that we have collected Personal Data from children without
+    verification of parental consent, we take steps to remove that information
+    from our servers.
+</p>
 
 <h2>Changes to This Privacy Policy</h2>
-<p>We may update our Privacy Policy from time to time. We will notify you of any changes by posting the new Privacy
-    Policy on this page.</p>
-<p>We will let you know via email and/or a prominent notice on our Service, prior to the change becoming effective and
-    update the "effective date" at the top of this Privacy Policy.</p>
-<p>You are advised to review this Privacy Policy periodically for any changes. Changes to this Privacy Policy are
-    effective when they are posted on this page.</p>
-
+<p>
+    We may update our Privacy Policy from time to time. We will notify you of
+    any changes by posting the new Privacy Policy on this page.
+</p>
+<p>
+    We will let you know via email and/or a prominent notice on our Service,
+    prior to the change becoming effective and update the "effective date" at
+    the top of this Privacy Policy.
+</p>
+<p>
+    You are advised to review this Privacy Policy periodically for any changes.
+    Changes to this Privacy Policy are effective when they are posted on this
+    page.
+</p>
 
 <h2>Contact Us</h2>
 <p>If you have any questions about this Privacy Policy, please contact us:</p>
 <ul>
     <li>By email: me@iamandrew.io</li>
```

Comment:
Shouldn't we replace this with one of our emails? As the owner of the repository and the domain it's hosted on, I'm fine with you adding `me@matteas.nz`.